plugins/flatpak/gs-flatpak.c

```diff
@@ -4062,9 +4057,16 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
-	if (!gs_flatpak_has_space_to_update (self, app, list)) {
-		g_debug ("Skipping automatic update for %s: not enough space on disk",
-			 gs_app_get_unique_id (app));
+	is_auto_update = gs_utils_app_is_auto_updating (app);
+	if (!gs_flatpak_has_space_to_update (self, app, list, is_auto_update)) {
+		if (is_auto_update)
+			g_debug ("Skipping automatic update for %s: not enough space on disk",
+				 gs_app_get_unique_id (app));
+		else
+			g_debug ("Skipping update for %s: not enough space on disk",
+				 gs_app_get_unique_id (app));
```

Comment:
How about:
`g_debug ("Skipping %s for %s: not enough space on disk", (is_auto_update ? " automatic update" : "update"),
		 gs_app_get_unique_id (app));`